sensors/lego_sensor_class.c

```diff
@@ -122,6 +123,12 @@
  *   error. The values are fixed point numbers, so check `decimals` to see if
  *   you need to divide to get the actual value.
  * .
+ * `text_value` (read-only)
```

Comment:
This is the documentation that will appear on the website. 

> is null terminated, and is limited
   to PAGE_SIZE-2 bytes to take into account the NULL terminator. We force
   a trailing linefeed on the return value.

This is assumed for all sysfs attributes and will be confusing for non-kernel hackers. Let's just omit the bit of text I quoted here.

The really important bits here are

* The value is sensor-specific.
* There MAY be more than one "value" and if so they are space-separated.